test/eunit/yaws_ls_tests.erl

```diff
@@ -0,0 +1,13 @@
+%% @doc `yaws_ls_tests' -- eunit tests for `yaws_ls'
+
+-module(yaws_ls_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+trim_test(Expected, Input, Len) ->
+    Enc = binary_to_list(unicode:characters_to_binary(Input)),
+    ExpectedEnc = binary_to_list(unicode:characters_to_binary(Expected)),
+    ExpectedEnc =:= yaws_ls:trim(Enc, Len).
+
+trim_test_() ->
+    [?_assert(trim_test("世界杯..&gt;", "世界杯足球赛", 11)),
```

Comment:
This line makes anything older than Erlang 17 very unhappy in the Travis builds.